libs/common/domain/src/lib/model/record/contact.model.ts

```diff
@@ -24,6 +25,39 @@ export const RoleValues = [
   'user', // Party who uses the resource
 ]
 
+export const RoleLabels = new Map<string, string>([
+  ['unspecified', marker('metadata.contactForResource.role.unspecified')],
```

Comment:
```suggestion
  ['unspecified', marker('domain.contact.role.unspecified')],
```

This kind of translation key would make more sense here IMO